package.json

```diff
@@ -240,6 +240,17 @@
     "url": "https://github.com/brave/brave-browser/issues"
   },
   "homepage": "https:/github.com/brave/brave-core#readme",
+  "config": {
+    "projects": {
+      "chrome": {
+        "dir": "src",
+        "tag": "92.0.4515.101",
```

Comment:
to check if this needs update after rebase

Reply:
`92.0.4515.101` at https://github.com/brave/brave-browser/blob/master/package.json#L234